### ISSUE TITLE
OutputDebugString event deadlock: Target stuck v0.9.10 ALPHA [da54fd1]

### ISSUE THREAD

**Issue by MohitSethi99** [EpicGamesExt/raddebugger#255]
Version: 0.9.10 ALPHA [41afe1a]
Target stuck and controls don't work

https://github.com/EpicGamesExt/raddebugger/assets/29519295/409b9900-ef13-4b2e-959e-731b8b05e6a2

**Comment by ryanfleury** [comment]
Can you describe, if possible, how you entered this state? Can you also send me the logs located in `%appdata%/raddbg/logs`?

**Comment by MohitSethi99** [comment]
I don't know if it makes sense but bug seems to be introduced in this commit: ffa802f5834e0af5663bd902c7ff07789dd85069

**Comment by MohitSethi99** [comment]
Tested extensively, the bug is introduced in this commit: da54fd12175dc594bef833763e8c1ed44bbec3c9

**Comment by MohitSethi99** [comment]
Waiting on this mutex I think:
<img width="1919" alt="image" src="https://github.com/EpicGamesExt/raddebugger/assets/29519295/220b40db-98d8-48d1-a720-6aec7e29d31e">

**Comment by MohitSethi99** [comment]
This might be a little more helpful:
![image](https://github.com/EpicGamesExt/raddebugger/assets/29519295/933a4638-c62c-4cf9-b89e-871c4c0ca5af)

Also, this string is the last logged event in the log file.

**Comment by MohitSethi99** [comment]
Here's the minimal repro:
OutputDebugString with `65409` bytes work but `65409 + 1` or more bytes will make the debugger go into this state.
```cpp
int main()
{
	// construct message with multiple lines
	char message[65409+1];
	int chunkSize = 128;
	int chunkCount = sizeof(message)/chunkSize-1;
	for(int i = 0; i < chunkCount; i++)
	{
		char ch = i == 0 ? '0' : '=';
		char* dst = &message[0] + chunkSize * i;
		memset(dst, ch, chunkSize-1);
		dst += chunkSize-1;
		memset(dst, '\n', 1);
	}
	char* dst = &message[0] + chunkSize * chunkCount;
	memset(dst, '=', chunkSize-1);
	message[sizeof(message) - 2] = '\n';
	message[sizeof(message) - 1] = 0;

	OutputDebugStringA(message);
	return 0;
}
```

**Comment by ryanfleury** [comment]
This should be fixed by fa4d0bc2628200bc207947935c2bb3d55469c5c3. Let me know if you run into any more issues with this.
